docs/htdocs/info/docs/tools/vep/script/vep_download.html

```diff
@@ -133,6 +133,8 @@ <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_R
 
     <li>New plugins (supported on CLI): </li>
       <ul>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/DeNovo.pm" rel="external">DeNovo</a> - identifies de novo variants in a VCF file</li>
```

Comment:
Can you add to the end: requires a pedigree (.ped) file

Reply:
@jamie-m-a , added!